### ISSUE TITLE
Shuffling

### ISSUE THREAD

**Issue by clowncracker** [jarnedemeulemeester/findroid#334]
Is shuffling possible in findroid?  I click on a series and I don't see a shuffle option anywhere.

**Comment by Arche151** [comment]
Nope, there is no shuffle feature unfortunately. I would love to have one. It's the only thing keeping me from switching to Findroid from the official Jellyfin app.

**Comment by polyzen** [comment]
Duplicate of #547
